arch/arm64/boot/dts/qcom/sdm660-xiaomi-clover-common.dtsi

```diff
@@ -590,6 +606,13 @@
 		drive-strength = <2>;
 		bias-pull-down;
 	};
+
+	gpio_hall_sensor_default: gpio-hall-sensor-default-state {
+		pins = "gpio75";
+		function = "gpio";
+		drive-strength = <2>;
+		bias-disable;
```

Comment:
It might have to be `bias-pull-up;` because downstream sets it this way.

They do this for gpio76 though, despite it being obviously wrong - schetics say that HALL_INT is gpio 75 and gpio76 is not even connected:

![image](https://github.com/user-attachments/assets/4887c085-8487-47d9-b361-8ffdedcfafce)

Reply:
And clover maintained downstream kernel also says bias-pull-down - https://github.com/kyasu/android_kernel_xiaomi_sdm660/commit/c7c9f86e3c0d84e13d8d8d4afd788cde121c6f90

Reply:
Yeah equal on tulip, I'm trying to remember why I blindly set those value. Maybe I blindly copy-pasted from platina.

Reply:
I did tests with
```
watch -n1 grep gpio75 /sys/kernel/debug/gpio
```
and flipping magnetic cover on/off and watching gpio 75 state in terminal. With `bias-disable` it sometimes sisn't change high/low state properly, with `bias-pull-down` it changes more reliably, always and faster.

Reply:
I open this https://github.com/sdm660-mainline/linux/pull/68

also `drive-strength` need to be set to 6 according to downstream

Reply:
bias-pull-down, also if downstream says bias-pull-up?